apps/api/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+COPY . .
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y pkg-config libxml2-dev libxmlsec1-dev libxmlsec1-openssl
+RUN pip install -r requirements.txt -r requirements-dev.txt
```

Suggestion: we can use the `--no-cache-dir` option to optimize the cache
> The --no-cache-dir option tells pip to not save the downloaded packages locally, as that is only if pip was going to be run again to install the same packages, but that's not the case when working with containers. ([FastAPI in Containers: Dockerfile](https://fastapi.tiangolo.com/deployment/docker/#dockerfile))

We also don't need all of the dev dependencies in the container as most are just for linting.
```suggestion
WORKDIR /code/
COPY requirements.txt .
RUN pip install --no-cache-dir -r requirements.txt "uvicorn[standard]"
COPY src/ src/
```